packages/editor/src/components/widgets/table/header/TableHeader.tsx

```diff
@@ -4,25 +4,36 @@ import type { ReactNode } from 'react';
 import Button from '../../button/Button';
 import { IvyIcons } from '@axonivy/editor-icons';
 
-export const TableHeader = (props: { colSpan: number; children?: ReactNode }) => (
-  <th className='table-column-header' colSpan={props.colSpan}>
+export const TableHeader = (props: { colSpan: number; children?: ReactNode; columnWidth?: number }) => (
+  <th className='table-column-header' colSpan={props.colSpan} style={props.columnWidth ? { width: `${props.columnWidth}px` } : {}}>
     {props.children}
   </th>
 );
 
-export function SortableHeader<TData>(props: { header: HeaderContext<TData, unknown>; name: string }) {
+export function SortableHeader<TData>(props: { header: HeaderContext<TData, unknown>; name: string; seperator?: boolean }) {
   const header = props.header;
   return (
-    <div className='column-sort'>
-      <span className='column-sort-label'>{props.name}</span>
-      <Button
-        className={header.column.getCanSort() ? 'column-sort-button' : ''}
-        aria-label={`Sort by ${props.name}`}
-        onClick={header.column.getToggleSortingHandler()}
-        data-state={header.column.getIsSorted() || 'unsorted'}
-        icon={header.column.getIsSorted() ? IvyIcons.Chevron : IvyIcons.Straighten}
-      />
-    </div>
+    <>
+      {props.seperator && (
+        <div
+          {...{
+            onMouseDown: header.header.getResizeHandler(),
+            onTouchStart: header.header.getResizeHandler(),
+            className: `resizer ${header.column.getIsResizing() ? 'isResizing' : ''}`
+          }}
```

Comment:
why wrap it in an object and then "unpack" it again?
is the following not working? 
```ts
<div onMouseDown={header.header.getResizeHandler()} onTouchStart={header.header.getResizeHandler()} className={`resizer ${header.column.getIsResizing() ? 'isResizing' : ''}`} />
```